src/main.rs

```diff
@@ -458,7 +452,7 @@ fn set_version(
                     use dialoguer::{theme::ColorfulTheme, Confirm};
 
                     let overwrite = Confirm::with_theme(&ColorfulTheme::default())
-                        .with_prompt(format!("Tag {} already exists. Overwrite?", tag_name))
+                        .with_prompt(format!("Tag {tag_name} already exists. Overwrite?",))
```

Comment:
`Overwrite?",` -> `Overwrite?"` (without trailing comma)

Reply:
This PR looked great, and I noticed just a small typo, which I went ahead and fixed before merging to keep things moving. Appreciate your contribution!